### PR TITLE
Feature/43 apply for task

### PR DESCRIPTION
**Pull request by mo3ni**
#43 
Notification

### CODE REVIEW

File: lib/login_page.dart [hunk 1]
```diff
@@ -5,6 +5,10 @@ import 'package:job_bazaar/auth.dart';
 import 'package:job_bazaar/signup_page.dart';
 import 'package:provider/provider.dart';
 
+// SingleChildScrollView(
```
Comment (Amal83) [inline lib/login_page.dart:8]: Can you, please, delete the commented lines, if it is not if they are not useable.

File: lib/screens/apply_task.dart [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  _ApplyTaskScreenState(this._task);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(_task.title),
+      ),
+      body: SingleChildScrollView(
+//        child: Column(
```
Comment (Amal83) [inline lib/screens/apply_task.dart]: same here, regarding to comments

File: lib/screens/apply_task.dart [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+//              decoration: BoxDecoration(
+//                color: Colors.orange,
+//                image: DecorationImage(
+//                  image: NetworkImage(""),
+//                  fit: BoxFit.cover
+//                ),
+//                borderRadius: BorderRadius.all(Radius.circular(75.0)),
+//              ),
+//            )
+//          ],
```
Comment (Amal83) [inline lib/screens/apply_task.dart]: same here, regarding to comments

File: lib/screens/apply_task.dart [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+                      try {
+                        FirebaseUser user = await FirebaseAuth.instance.currentUser();
+
+                        print(_task.toString());
+                        
+                        await Firestore.instance.collection('tasks/${_task.id}/applicants').document(user.uid).setData({'status': 'interrested'});
+
+                        // _task.
+
+                        // await Firestore.instance.collection('tasks')
```
Comment (Amal83) [inline lib/screens/apply_task.dart]: same here, regarding to comments

Comment (elvarlax) [reply lib/screens/apply_task.dart]: Remove comments, commit and create a new pull request

File: lib/login_page.dart (hunk 1 above)
Comment (mo3ni) [reply lib/login_page.dart:8]: sure

File: lib/screens/apply_task.dart (hunk 3 above)
Comment (mo3ni) [reply lib/screens/apply_task.dart]: done

**Review by atila29 — APPROVED** [review]
Create another issue & branch to fix the button.

**Pull request merged**